utils/cronjobs_osgeo_lxd/hugo_clean_and_update_job.sh

```diff
@@ -3,31 +3,25 @@
 # 2020-2024, Markus Neteler
 # deploy updated web site from github repo
 
-# preparation
+# preparations:
 #  sudo chown -R neteler.users /var/www
+# get grass-website repo
+#  cd ~/
+#  git clone https://github.com/OSGeo/grass-website.git
+####
 
-# 0.  change into local git repo copy
-#     cd ~ ; git clone https://github.com/OSGeo/grass-website.git
-#     cd grass-website
-# 1.  update local repo from github
-# 2.  rm previously built pages in local git repo copy
-# 3.  build updated pages with hugo
-# 4.  create tmp target web directory
-# 5.  copy over updated pages to tmp target web directory
-# 6.  rm previously deployed web pages while not deleting src code files in their adjacent directory (careful!)
-# 7.  copy over updated pages from tmp target web directory to server target web directory
-# 8.  links src code dir content into now deployed web directory
-# 9.  rm tmp target web directory
-# 10. restore linked src file timestamps from their original source time stamps
+# 1. change into local git repo copy
+# 2. update local repo from github
+# 3. build updated pages with hugo into clean directory
+# 4. rsync over updated pages to target web directory, deleting leftover files
+# 5. generate links from src code directory content into web directory
+# 6. restore timestamps of links from their original time stamps in src directory
 
 cd /home/neteler/grass-website/ && \
    git pull origin master && \
    rm -rf /home/neteler/grass-website/public/* && \
    nice /home/neteler/go/bin/hugo && \
-   mkdir /var/www/html_new && \
-   \cp -rp /home/neteler/grass-website/public/* /var/www/html_new/ && \
-   rm -fr /var/www/html/* && \
-   \mv /var/www/html_new/* /var/www/html/ && \
+   rsync -a --delete /home/neteler/grass-website/public/ /var/www/html/ && \
    ln -s /var/www/code_and_data/* /var/www/html/ && \
-   rmdir /var/www/html_new && \
-   (cd /var/www/html/ ; /home/neteler/bin/fix_link_timestamp.sh .)
+   (cd /var/www/html/ ; /home/neteler/bin/fix_link_timestamp.sh .) && \
```

Comment:
Not directly related to this PR, but is the timestamp fixing needed? I don't see the script in this repo. Maybe you have a comment there?

Reply:
It has been my (personal) policy to not damage timestamps when copying over files. I tried to carefully keep the server timestamps since the late 1990s :-) Random example: https://grass.osgeo.org/grass42/source/

The [fix_link_timestamp.sh](https://github.com/neteler/script_collection/blob/master/filename_tricks/fix_link_timestamp.sh) script I can add to this PR if desired.

Reply:
Lovely. Can you put there a comment that his makes the time stamps of the symlinks the same as time stamps of the original files? (If that's what this is doing.)

Reply:
Time stamp restore function included in 9963b614d7fcb1a0dfad68cb1baca07a1941d5ec